lib/rollbax/client.ex

```diff
@@ -40,12 +40,14 @@ defmodule Rollbax.Client do
   end
 
   def handle_cast({:emit, event}, %{enabled: :log} = state) do
-    {level, message, time, meta} = event
+    {level, exception, stacktrace, time, meta} = event
     Logger.info [
-      "(Rollbax) registered report:", ?\n, message,
+      "(Rollbax) registered report:", ?\n, exception.message,
```

Comment:
We should likely print the exception here as well (instead of just the message). For example, `[..., ?(, inspect(exception.__struct__), ") ", exception.message, ...]`.